content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -345,7 +345,7 @@ either be a snapshot file from a previous backup operation, or from a remaining
 When restoring the cluster, use the `--data-dir` option to specify to which folder the cluster should be restored:
 
 ```shell
-etcdutl --data-dir <data-dir-location> snapshot restore snapshot.db
+etcdctl --data-dir <data-dir-location> snapshot restore snapshot.db
```

Comment:
I think the existing content is correct.

Reference:

- https://etcd.io/docs/v3.6/op-guide/recovery/#restoring-a-cluster
- https://github.com/etcd-io/etcd/blob/main/etcdutl/README.md

Reply:
See also https://github.com/kubernetes/website/issues/30215

Reply:
It would be better if we could mention both approaches on this task page or mention Notes to clarify that both commands are correct so that folks won't get confused in the future.

Reply:
As mentioned in the following [comment](https://github.com/kubernetes/website/issues/30215#issuecomment-950644306) we can use tabs to mention both commands.

Reply:
Have a look at the following PR also: #44050

Reply:
Thanks a lot for the clarification, I was misled by the failed etcdutl command